supertokens_python/logger.py

```diff
@@ -0,0 +1,71 @@
+import json
+import logging
+import logging.config
+from datetime import datetime
+from os import getenv
+from typing import Any, Dict, Optional
+
+from constants import VERSION
+
+
+class LoggerCodes:
+    ApiResponse = 1
```

Comment:
Since this is an enum, you should make it `API_RESPONSE`?

Reply:
Valid point.